3.7/aql/functions-miscellaneous.md

```diff
@@ -235,6 +238,10 @@ The function can also be used with a single parameter *id* as follows:
 ```js
 DOCUMENT("users/john")
 DOCUMENT( [ "users/john", "users/amy" ] )
+
+DOCUMENT( @id ) // bind vars: {"id": "users/john"}
+DOCUMENT( @ids ) // bind vars: {"id": ["users/john", "users/amy"]}
```

Comment:
The bind var should be `ids` not `id` for the multiple users example. Otherwise, looks good to me.